src/pf-dropdown/pf-dropdown.spec.js

```diff
@@ -24,7 +24,34 @@ describe("Patternfly Dropdown Component Test", function () {
 
     customElement.addEventListener('hidden.bs.dropdown', function () {
       expect(button.parentNode.classList.contains('open')).toBe(false);
-      done();
     }, false);
   });
+
+  it("disabled button should not open dropdown", function () {
+    var button = customElement.querySelector('.btn');
+    button.classList.add('disabled');
+    customElement.toggle();
+    expect(button.parentNode.classList.contains('open')).toBe(false);
+    button.dispatchEvent(new MouseEvent('click'));
+    expect(button.parentNode.classList.contains('open')).toBe(false);
+  });
+
+  it("should select an element", function () {
+    var buttonText = customElement.querySelector('.btn');
+    var item = customElement.querySelector('ul.dropdown-menu > li:nth-child(2)');
+    expect(buttonText.innerText).toBe('Dropdown');
+    expect(item.innerText).toBe('Item 1');
+  });
+
+  it("should not click disabled item", function () {
```

Comment:
Instead of creating an event and dispatching yourself, I was expecting to use a mouse click. For example:
item.click();

You can add your own event listener to verify the selection.